fusell.py

```diff
@@ -263,7 +263,7 @@ class fuse_ctx(ctypes.Structure):
 c_stat_p = ctypes.POINTER(c_stat)
 fuse_file_info_p = ctypes.POINTER(fuse_file_info)
 
-FUSE_SET_ATTR = ('st_mode', 'st_uid', 'st_gid', 'st_size', 'st_atime', 'st_mtime')
+FUSE_SET_ATTR = ('st_mode', 'st_uid', 'st_gid', 'st_size', 'st_atime', 'st_mtime', None, 'st_atime_now', 'st_mtime_now', 'st_ctime')  # Flag #6 (FATTR_FH) is never received by setattr
```

Comment:
can you change this to a multi line constant and put your comment on the `None` flag?